BUILD.md

```diff
@@ -117,35 +117,34 @@ and run Conduit:
 ## Comprehensive
 
 This configuration builds all Conduit components in Docker images, and deploys
-them onto Minikube. This configuration also builds and installs a `conduit`
-executable onto the local system. This setup most closely parallels our
-recommended production installation, documented at
-https://conduit.io/getting-started/.
+them onto Minikube. This setup most closely parallels our recommended production
+installation, documented at https://conduit.io/getting-started/.
 
-These commands assume working [Go](https://golang.org) and
-[Minikube](https://github.com/kubernetes/minikube) environments.
+These commands assume a working
+[Minikube](https://github.com/kubernetes/minikube) environment.
 
 ```bash
-# ensure all go dependencies are in vendor
-dep ensure && dep prune
-
-# verify cluster (minikube) status
-bin/go-run cli check
-
 # build all docker images, using minikube as our docker repo
-DOCKER_FORCE_BUILD=1 DOCKER_TRACE=1 bin/mkube bin/docker-build latest
+# this command will also place conduit executables in target/cli/
+DOCKER_TRACE=1 bin/mkube bin/docker-build
+
+# note: depending on your OS, use one of these three conduit executables:
+# target/cli/darwin/conduit
+# target/cli/linux/conduit
+# target/cli/windows/conduit
 
 # install conduit
-bin/go-run cli install --version latest | kubectl apply -f -
+target/cli/darwin/conduit install --version $(bin/root-tag) | kubectl apply -f -
```

Comment:
Rather than asking folks to execute from the target directory, it might be more user-friendly to provide a `bin/conduit` script that does this for them. Something like:

```
#!/bin/bash

set -eu

system=$(uname -s)

if [[ $system = "Darwin"* ]]; then
  exec target/cli/darwin/conduit $@
elif [[ $system = "Linux"* ]]; then
  exec target/cli/linux/conduit $@
elif [[ $system = "Windows"* ]]; then
  exec target/cli/windows/conduit $@
else
  echo "unknown system: $system" >&2
  exit 1
fi
```

(not sure if those are the right system names to use, especially for windows)

But if this seems reasonable am happy to add it in a follow-up PR.

Reply:
Yeah, I agree we could do better than asking folks to run something like `target/cli/darwin/conduit`. I would also love to decrease the number of scripts in `./bin/`, but I think what you're proposing would be quite useful and worth the trade off.